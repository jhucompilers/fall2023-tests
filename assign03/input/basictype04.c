int main(void) {
  long l1;
  signed long l2;
  unsigned long l3;

  l1 = -9223372036854775806;
  l2 = -9223372036854775805;
  l3 = 9223372036854775804;

  return 0;
}
