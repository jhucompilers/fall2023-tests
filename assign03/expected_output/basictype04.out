0|main|function|function () returning int
2|l1|variable|long
2|l2|variable|long
2|l3|variable|unsigned long
